web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2372,7 +2395,20 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
     }
   }
 
-  private def buildAndStoreComet[T <: LiftCometActor](newCometFn: (CometCreationInfo)=>Box[T])(creationInfo: CometCreationInfo): Box[T] = {
+  /**
+   * Need to add your own CometActor to a Session/Page? You might 'cause creating
+   * a CometActor programmatically within a snippet may be between that referring
+   * to a CometActor by class name.
+   *
+   * This is dangerous stuff, so use with caution.
+   *
+   * @param newCometFn a Function that takes `CometCreateInfo` and returns a CometActor
+   *                   instance
+   * @param creationInfo the info to send to the above function
+   * @tparam T A LiftCometActor or better
+   * @return The created, registered, etc. CometActor
+   */
+  def buildAndStoreComet[T <: LiftCometActor](newCometFn: (CometCreationInfo)=>Box[T])(creationInfo: CometCreationInfo): Box[T] = {
```

Comment:
This is like way late for me to be asking this, @dpp, but what was the reasoning behind exposing this instead of just using `S.findOrCreateComet[T]`, which also handles ensuring that names are unique in the session? Wanted to drop a note about it in the release notes.